netbox/provider.go

```diff
@@ -40,12 +40,14 @@ func Provider() *schema.Provider {
 		DataSourcesMap: map[string]*schema.Resource{
 			"netbox_cluster":          dataSourceNetboxCluster(),
 			"netbox_cluster_group":    dataSourceNetboxClusterGroup(),
+                        "netbox_cluster_type":     dataSourceNetboxClusterType(),
```

Comment:
This looks wrongly indented. Did you run `make fmt`?

Reply:
Apologies, updated the PR!